lib/membrane/core/parent/child_life_controller/link_handler.ex

```diff
@@ -195,4 +189,14 @@ defmodule Membrane.Core.Parent.ChildLifeController.LinkHandler do
     ])
     ~> {&1, state}
   end
+
+  defp send_linking_finished(links) do
+    endpoints =
+      links
+      |> Enum.flat_map(&[&1.from, &1.to])
+      |> Enum.uniq()
+      |> Enum.reject(&(&1.child == {Membrane.Bin, :itself}))
+
+    Bunch.Enum.try_each(endpoints, &Message.call(&1.pid, :linking_finished))
```

Comment:
Why it isn't part of the pipe above?

Reply:
It's more readable this way IMO

Reply:
I really don't see any benefit besides naming endpoints, but you can to this in pipe
```suggestion
      links
      |> Enum.flat_map(&[&1.from, &1.to])
      |> Enum.uniq()
      |> Enum.reject(&(&1.child == {Membrane.Bin, :itself}))
      |> Bunch.Enum.try_each(fn endpoint -> Message.call(endpoint.pid, :linking_finished) end)
```
(commiting above from GH won't work)
Anyway, it's a matter of personal preference, so do as you wish